src/main/java/games/strategy/engine/data/properties/FileProperty.java

```diff
@@ -167,7 +166,7 @@ public String getDescription() {
         return Arrays.toString(acceptableSuffixes);
       }
     });
-    final int rVal = fileChooser.showOpenDialog(MainFrame.getInstance());
+    final int rVal = fileChooser.showOpenDialog(null);
```

Comment:
Not a big deal, but I just wanted to point out that using a `null` parent frame will center child windows to the screen rather than the TripleA application.  This only becomes an issue when you're running multiple (non-maximized) instances of TripleA.

I typically do this when testing network compatibility.  I'll open two TripleA applications and run them side-by-side on my single-monitor system.  It gets confusing when one instance pops up a window and it's centered relative to the screen rather than the originating instance.  Then I have to spend a few seconds figuring out which application it is associated with (usually clear from the title bar).

With that said, I'm wondering if we should be striving to use a non-`null` frame window wherever possible instead of going in the other direction?  Should the `GameRunner#mainFrame` field be exposed through a public static method, e.g. `GameRunner#getMainFrame()`?  I suspect there will be other changes in this PR whereby the main frame reference is replaced with `null`.